h/models/token.py

```diff
@@ -39,8 +39,9 @@ class Token(Base, mixins.Timestamps):
     #: A NULL value in this column indicates a token that does not expire.
     expires = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
 
-    def __init__(self, userid):
+    def __init__(self, userid, expires=None):
```

Comment:
Hmm. This is fine, but we should probably fix this properly -- i.e. not override the method signature of the default sqlalchemy constructor, and update uses of `Token(userid)` to do `Token(userid=userid)` instead.